pb_plugins/dcsdkgen/utils.py

```diff
@@ -100,9 +99,6 @@ def get_template_env(_searchpath):
         searchpath=_searchpath))
 
     # Register some functions we need to access in the template
-    _template_env.globals.update(
-        letter_case_to_delimiter=letter_case_to_delimiter,
-        indent=jinja_indent,
-        remove_subscribe=remove_subscribe)
+    _template_env.globals.update(indent=jinja_indent)
```

Comment:
+1

We should also remove code for `letter_case_to_delimiter` and `remove_subscribe`, right?

Reply:
Oh, right, I'll check that!